experiments/functional/jiva-csi/jiva-csi-busybox/test_vars.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+# Test specific parameters
+
+action: "{{ lookup('env','ACTION') }}"
+
+app_label: "{{ lookup('env','APP_LABEL') }}"
+
+app_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+
+operator_ns: 'openebs'
+
+pvc_name: "{{ lookup('env','PVC') }}"
+
+storage_capacity: "{{ lookup('env','CAPACITY') }}"
+
+storage_class: "{{ lookup('env','PROVIDER_STORAGE_CLASS') }}"
```

Comment:
How replicasc will be obtained @somesh2905

Reply:
@gprasath this is the storage class created by jiva csi provisioner and it will take care of replicasc itself.

Reply:
done